### PR TITLE
Correct slash and backslash

### PR DESCRIPTION
**Pull request by cyrichardson**

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-1f50400fc0/docs/style-guide/).

**Comment by cyrichardson** [comment]
@RackerKate Please review when you have time. This doesn't include much for error messages, but it does correct slash versus backslash, I think. Thanks!

**Comment by cyrichardson** [comment]
@RackerKate Let me try to fix some disappearing backslash characters. I'll ping you again when I've fixed those up.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-9075229b84/docs/style-guide/). :bow:

**Comment by cyrichardson** [comment]
@RackerKate Ready for your review - next week is fine, if you can work it in then. Thanks!
(Properly escaping the backslash character in restructured text was the problem all along.)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-e516f841fd/docs/style-guide/).

File: style-guide/terminology/alphabetical-terms.rst [hunk 1]
```diff
@@ -2495,12 +2496,12 @@ parts of speech:
        See also *as, since (restricted use)*.
    * - slash
      -
-     - Use *slash* to refer to the / character. Use *backslash* to refer to
-       the character. Don't use *slash mark*.
+     - Use *slash* to refer to the / character, and use *backslash* to refer
+       to the \\ character. Don't use *slash mark*.
```
[the remark this answers is not in the document]
Comment (cyrichardson) [reply style-guide/terminology/alphabetical-terms.rst]: Local tox build and preview link show the backslash character on my laptop

**Comment by cyrichardson** [comment]
@RackerKate The backslash character is displaying properly in my local tox build, preview, and published docs. When you have time, please check the published docs on your computer to see if they are displaying properly for you now. (Are you by chance on a Windows computer?)

https://developer.rackspace.com/docs/style-guide/terminology/alphabetical-terms/
https://developer.rackspace.com/docs/style-guide/style/symbols/

**Comment by RackerKate** [comment]
Yes, I see them in the published docs!

**Comment by cyrichardson** [comment]
Woot! Thanks for checking! Gremlins!

**Pull request merged**